languages/zh-CN.yml

```diff
@@ -45,6 +45,8 @@ page:
 footer:
   powered: "由 %s 强力驱动"
   theme: 主题
+  site_pv: 总访问量
```

Comment:
@ivan-nginx  

The field name is not correct.

It should be `total_views` as the same with `en.yml`

And the `site_pv` should be `total_visitors`

Reply:
@wafer-li seems no need to manual change the `zh-CN.yml`, any language res files will be fixed by the crowdin.

Reply:
OK, then leave it to crowdin.